x-pack/solutions/search/test/functional_solution_sidenav/config.ts

```diff
@@ -25,7 +25,10 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     testFiles: [require.resolve('.')],
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
-      serverArgs: [...functionalConfig.get('kbnTestServer.serverArgs')],
+      serverArgs: [
+        ...functionalConfig.get('kbnTestServer.serverArgs'),
+        '--feature_flags.overrides.searchSolution.gettingStartedEnabled=true',
```

Comment:
This might be problematic if these tests are run against a real cloud environment since they become not environment-agnostic. But I don't know if this config runs!